swmm-toolkit/LICENSE.md

```diff
@@ -0,0 +1,159 @@
+
```

Comment:
@michaeltryby This is super interesting!

Reply:
Since the wrapper code is machine generated it is unlikely that copyright claims on it would be valid. So I opted for placement in public domain. Because it wraps SWMM library there is a place to recognize third party library licensing. So that is where OWA SWMM license would go.